units/user-config.target

```diff
@@ -11,3 +11,9 @@ After=user-cloudinit@var-lib-coreos\x2dinstall-user_data.path
 
 Requires=user-cloudinit-proc-cmdline.service
 After=user-cloudinit-proc-cmdline.service
+
+Requires=user-config-vmware-tools.service
```

Comment:
Using the conditions would be inexpensive (I forgot that was there). Unfortunately, there is a fair amount of coupling between this repo and what is actually shipped in the OS. I'm not ready to sit down and figure out how PXE on VMware should work just yet, so can you pull this into a separate PR? Then I can come back to it in a bit and think it through.